test/test_jit_cuda_fuser.py

```diff
@@ -4588,12 +4597,77 @@ def test_nvfuser_correctness(self, device, dtype, op):
 
             self.assertEqual(ref, val, exact_layout=True)
 
+        # Note: Clearing CU after NVFuser tests
         # https://github.com/pytorch/pytorch/issues/35600
         # each torch.jit.trace adds state to the _python_cu compilation unit
         # since this test traces a lot of functions, out-of-memory can occur
         # if the CU is not cleared.
         torch.jit._state._python_cu.drop_all_functions()
 
+    @slowTest
+    @unittest.skipIf(not RUN_NVFUSER, "requires CUDA")
+    @unittest.skipIf(GRAPH_EXECUTOR != ProfilingMode.PROFILING,
+                     "Requires fusion optimization pass to be effective")
+    @ops(op_db, allowed_dtypes=(torch.float16, torch.bfloat16, torch.float32,
```

Comment:
What happens if nvFuser doesn't support the operator?

Reply:
We'll still run the test, but the operator won't be fused by nvfuser (so it'll just run the usual eager implementation of the op)

Reply:
That kinda seems excessive, right? Not the end of the world. In the future it'd be nice to know which operators are supported by nvFuser so we don't excessively test

Reply:
Is there a good way to keep an up to date list of ops supported by NVFuser? I imagine that it would be pretty easy to forget to update the list when NVFuser adds new ops

Reply:
Yeah that's a good concern -- I don't know offhand